Makefile

```diff
@@ -41,6 +41,32 @@ setup:
 	cargo install elf2tab --version 0.4.0
 	cargo install stack-sizes
 
+# Downloads a version of QEMU with patches needed to support Tock on HiFive.
+# Builds QEMU and installs it into build/.
+.PHONY: setup-qemu
+setup-qemu:
+	mkdir -p build
+	cd build                                              && \
+	git clone https://github.com/alistair23/qemu.git         \
```

Comment:
So Tock already has support for setuping QEMU, can we use that?

Reply:
D'oh, I missed that Tock has its own QEMU. I just pushed a commit that uses Tock's QEMU rather than our own QEMU.